ChangeLog.d/pkg-config-files-addition.txt

```diff
@@ -0,0 +1,2 @@
+Features
+    * Add pc files for pkg-config. eg.) pkg-config --cflags --libs (mbedtls|mbedcrypto|mbedx509)
```

Comment:
The CI is complaining because this line is too long. The changelog entry from the original PR would be fine

```
   * Add pc files for pkg-config, e.g.:
     pkg-config --cflags --libs (mbedtls|mbedcrypto|mbedx509)
```

Reply:
> The CI is complaining because this line is too long. The changelog entry from the original PR would be fine
> 
> ```
>    * Add pc files for pkg-config, e.g.:
>      pkg-config --cflags --libs (mbedtls|mbedcrypto|mbedx509)
> ```

Its a cherry-pick, it's exactly the same as the original PR

Reply:
> > The CI is complaining because this line is too long. The changelog entry from the original PR would be fine
> > ```
> >    * Add pc files for pkg-config, e.g.:
> >      pkg-config --cflags --libs (mbedtls|mbedcrypto|mbedx509)
> > ```
> 
> Its a cherry-pick, it's exactly the same as the original PR

[Update ChangeLog.d/pkg-config-files-addition.txt](https://github.com/Mbed-TLS/mbedtls/pull/8691/commits/fff51ceccdb8b80553ce3d38e266c21e24872e9c)